tests/CMakeLists.txt

```diff
@@ -47,7 +47,8 @@ add_custom_target(check-full COMMAND CTEST_OUTPUT_ON_FAILURE=1 ${CMAKE_CTEST_COM
 
 # Reference asn1c libraries
 add_asn1c_lib(config asn1-reference/common asn1-reference/config/cryptosource asn1-reference/config/tlsconfig)
-list(APPEND TEST_LIBS config_asn1)
+add_asn1c_lib(ipuptane asn1-reference/common asn1-reference/uptane/ipuptane_message)
+list(APPEND TEST_LIBS config_asn1 ipuptane_asn1)
```

Comment:
Originally, it was intended to have only one asn1c_lib in the code base, because the common generated C files could not link with each others. Also, cmake now outputs twice the garbage.

Reply:
@lbonn Yes, might work that way. I'll change FindAsn1.cmake a bit and we'll figure out if that works.